samples/drivers/display/boards/mimxrt1170_evk_cm7.conf

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright (c) 2022, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_HEAP_MEM_POOL_SIZE=4194304
```

Comment:
Taking in the account previous commit, why are you adding this overlay?

Reply:
This is needed as we require more memory to allocate the frame buffers allocated in `driver_mcux_elcdif.c`

Reply:
This is because `CONFIG_HEAP_MEM_POOL_SIZE=16384` is defined in `samples/drivers/display/prj.conf`. This is eliminating the default value defined in the boards `Kconfig.defconfig` file. I would prefer not to have this in the overlay but I did not want to impact other platforms display testing.

Reply:
Right. Do you need to set it in [boards/arm/mimxrt1170_evk/Kconfig.defconfig](https://github.com/zephyrproject-rtos/zephyr/pull/36875/commits/57132abf74aa5d66b97c7c794b547727c35f4912#diff-03a9c4ae2d435ebefab2f185a68774643ae7cfd58b36a3a62cf9e4473dc33470) then? Because as I see it LVGL sample does not need it.

Reply:
This board needs the extra `HEAP`, so I would like to keep it in the board file so that application developers do not have to worry about this.

Reply:
it looks like the setting is adjusted in two places, defconfig and here. why isn't defconfig sufficient?

Reply:
The tests `proj.conf` file overrides the defconfig value. `samples/drivers/display/prj.conf`

Reply:
boards/arm/mimxrt1170_evk/Kconfig.defconfig:

```
# Memory from the heap pool is used to allocate display buffers.
# Adjust HEAP_MEM_POOL_SIZE.
config HEAP_MEM_POOL_SIZE
	default 4194304
```

samples/drivers/display/boards/mimxrt1170_evk_cm7.conf
```
CONFIG_HEAP_MEM_POOL_SIZE=4194304
```

ie both set `4194304`

Reply:
`samples/drivers/display/prj.conf` overwrites the value set inside `boards/arm/mimxrt1170_evk/Kconfig.defconfig`. Hence this was added in the `samples/drivers/display/boards/mimxrt1170_evk_cm7.conf` file.